### ISSUE TITLE
Crash on opening folder

### ISSUE THREAD

**Issue by bignose-debian** [zeapo/Android-Password-Store#202]
In recent versions of Android Password Store, when I open the app:

* The top level of folders is presented.
* Select a folder to open.
  * The app crashes (“Unfortunately, Password Store has stopped.”).

Password Store version 1.2.0.48
Cyanogenmod version 12.1 (Android version 5.1.1)

**Comment by Cardridge** [comment]
This has been fixed in 9c23263. The corresponding version just isn't available on F-Droid yet.

**Comment by falsechicken** [comment]
Is there an ETA on when it will be available in F-Droid? I guess they only build releases or something?

**Comment by wongma7** [comment]
F-Droid is having issues building the fix release
https://f-droid.org/wiki/page/com.zeapo.pwdstore/lastbuild_71 . I opened an
issue here (but in the wrong repo)
https://gitlab.com/fdroid/fdroiddata/issues/487 and it's being worked on.
Maybe I can ask them to remove the broken version since this is quite
urgent, I don't know, I'll try to figure out something tonight.

Sorry about this, this bug I introduced is bad enough but I didn't foresee
this long a delay in getting the fix out, and I haven't been following it
too closely

To workaround: I know downgrading can be a hassle, but that's probably the
best; glad some people have done that. Also I'm not sure but maybe you
could use search bar to navigate if you know the name of the pass you want,
lol
